WEB/Scraper/Scraper.py

```diff
@@ -0,0 +1,10 @@
+from twitterscraper import query_tweets
```

Comment:
I think we we did some good work looking at twitter data at the meetup tonight. Let's look into the twitter api for gathering twitter data instead of the twitterscraper library.

We'll need the geo data.